app/main.py

```diff
@@ -1,8 +1,34 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class, clean_mark, brand):
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center, clean_power, average_rating, count_of_ratings):
+        self.distance_from_city_centre = distance_from_city_center
```

Comment:
There is a typo in the variable name. `distance_from_city_centre` should be `distance_from_city_center` to match the parameter name in the `__init__` method.